### DISCUSSION TITLE
Is there any way to use own board map on web builder?

### DISCUSSION THREAD

**Discussion by ashokmanker** [grblHAL/core#495] [General]
I am using mks Tinybee board. I have complied grblhal sometimes and flashed. But this time I can't. Web builder is simple way to compile without much hassle. I am using custom boardmap. Since I want o use 6 limit switches and few custom pin assignment with option for laser. If there is a way to include custom boardmap. Then it will be much easier for everyone.

**Comment by terjeio** [comment]
Sorry, there is no way and currently I do not plan to add it.
